data/pb/BillingService/TaxHeadMaster.json

```diff
@@ -2171,6 +2171,16 @@
       "order": "7",
       "isRequired": false
     },
+    {
```

Comment:
Code should be in caps and you have already added same code in taxHead master. Duplicate codes are not allowed.

Reply:
Okay